mgmt_cli_test.py

```diff
@@ -112,10 +112,11 @@ def _run_cmd_with_retry(self, executor, cmd, retries=10):
 
     def install_awscli_dependencies(self, node):
         if node.distro.is_ubuntu or node.distro.is_debian:
-            cmd = dedent("""
-            apt update
-            apt install -y python3-pip
-            pip install awscli==1.18.140
+            pip_break_system_packages = "--break-system-packages" if node.distro.is_ubuntu24 else ""
```

Comment:
One more option is to use the environment variable for it, and you don't need a condition

Reply:
Agree, looks way better without this condition, fixed.
Retesting: https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/manager-master/job/ubuntu22-sanity-test/38/